### PR TITLE
Clarify app integration

### PR DESCRIPTION
**Pull request by jonkjenn**

### CODE REVIEW

File: how-it-works/vipps-login-apps-howitworks.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# How Login works from apps
+
+<!-- START_COMMENT -->
+
+ℹ️ Please use the website:
+[Vipps MobilePay Technical Documentation](https://vippsas.github.io/vipps-developer-docs/docs/APIs/login-api).
+
+<!-- END_COMMENT -->
+
+Vipps Login app integrations uses the browser for performing logins. See [Vipps Login in browser: How It Works](vipps-login-api-howitworks.md).
```
Comment (rebekaburnett) [inline how-it-works/vipps-login-apps-howitworks.md:21]: I think I would delete this page, unless you are going to add more to it in the future.

Comment (cloveras) [reply how-it-works/vipps-login-apps-howitworks.md:21]: It _is_ super short..? 🧐

Comment (jonkjenn) [reply how-it-works/vipps-login-apps-howitworks.md:21]: I think we could make things clearer for merchants if we actually did describe the complete flow here. It's only partially true that it's the same as the one we link too.

File: api-guide/app-integration.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ END_METADATA -->
 
 # App integration
 
+App integrations are currently based on using browsers. An app integration flow without the use of browsers is on the backlog but prioritization is uncertain, please contact us if you're interested.
```
[the remark this answers is not in the document]
Comment (tinolazreg) [reply api-guide/app-integration.md:14]: We can keep "App integrations are currently based on using browsers.", but remove the rest.

**Review by tinolazreg — APPROVED** [review]
👍

**Pull request merged**